<launch>

<node pkg="feature_frontend" type="maskGenerater" name="maskGenerater" output="screen">
    <param name="camera_model" type="string" value= "/home/nvidia/ws/src/2_global_estimator_vins_stereo/cfg/car_small/cam_left.yaml"/>
    <param name="output_image" type="string" value= "/home/nvidia/ws/src/2_global_estimator_vins_stereo/cfg/car_small/cam_mask.bmp"/>
    <param name="maxFOV" type="int" value= "280"/>
    <param name="minFOV" type="int" value= "0"/>
</node>

</launch>
